.github/quarkus-github-bot.yml

```
@@ -225,6 +225,9 @@ triage:
         - integration-tests/spring-
     - labels: [env/windows]
       titleBody: "windows"
+    - labels: [env/m1]
+      titleBody: "\\b[mM]1\\b"
```

```suggestion
      titleBody: "\\bm1\\b"
```
The match is case insensitive already.